docs/usage.md

```diff
@@ -12,59 +12,44 @@ You will need to create a samplesheet file with information about the samples in
 --input <path to samplesheet file>
 ```
 
-An example sample sheet structure is shown below. This defines two target experimental groups for the histone marks h3k27me3 and h3k4me3 with two biological replicates per group. Each antibody target also had an IgG control performed alongside. The two IgG experiments are configured as biological replicates with the `igg` group keyword with a unique control group assignment. The target experiments are then assigned to the igg control group using the `control_group` column.
+An example sample sheet structure is shown below. This defines two target experimental groups for the histone marks h3k27me3 and h3k4me3 with two biological replicates per group. Each antibody target also has an IgG control. The two IgG experiments are configured as biological replicates in the same group named `igg_ctrl`. They are assigned as controls to the two other groups using the last `control` column. If there are an equal number of replicates assigned to the samples from the control group as is the case below, the IgG controls will automatically be assigned to the same replicate number. If there is a mismatch then the first rpelicate of the control group will be assigned to all.
 
 ```bash
-group,replicate,control_group,fastq_1,fastq_2
-h3k27me3,1,1,READ1_FASTQ.gz,READ2_FASTQ.gz
-h3k27me3,2,1,READ1_FASTQ.gz,READ2_FASTQ.gz
-h3k4me3,1,2,READ1_FASTQ.gz,READ2_FASTQ.gz
-h3k4me3,2,2,READ1_FASTQ.gz,READ2_FASTQ.gz
-igg,1,1,READ1_FASTQ.gz,READ2_FASTQ.gz
-igg,2,2,READ1_FASTQ.gz,READ2_FASTQ.gz
+group,replicate,fastq_1,fastq_2,control
+h3k27me3,1,READ1_FASTQ.gz,READ2_FASTQ.gz,igg_ctrl
+h3k27me3,2,READ1_FASTQ.gz,READ2_FASTQ.gz,igg_ctrl
+h3k4me3,1,READ1_FASTQ.gz,READ2_FASTQ.gz,igg_ctrl
+h3k4me3,2,READ1_FASTQ.gz,READ2_FASTQ.gz,igg_ctrl
+igg_ctrl,1,READ1_FASTQ.gz,READ2_FASTQ.gz,
+igg_ctrl,2,READ1_FASTQ.gz,READ2_FASTQ.gz,
 ```
```

Comment:
A naive user question ... are the replicates merged before analysis ?

Reply:
Yes if they are exactly the same id + rep number